pymtl3/stdlib/ifcs/SendRecvIfc.py

```diff
@@ -71,33 +81,16 @@ def connect( s, other, parent ):
 # SendIfcRTL
 #-------------------------------------------------------------------------
 
-class SendIfcRTL( Interface ):
+class SendIfcRTL( CallerIfcRTL ):
 
   def construct( s, Type ):
-
-    s.msg = OutPort( Type )
-    s.en  = OutPort( int if Type is int else Bits1 )
-    s.rdy =  InPort( int if Type is int else Bits1 )
-
-    s.MsgType = Type
-
-  def line_trace( s ):
-    try:
-      trace_len = s.trace_len
-    except AttributeError:
-      s.trace_len = len( "{}".format( s.MsgType() ) )
-      trace_len = s.trace_len
-
-    return enrdy_to_str( s.msg, s.en, s.rdy, trace_len )
-
-  def __str__( s ):
-    return s.line_trace()
+    super().construct( en=True, rdy=True, MsgType=Type, RetType=None )
```

Comment:
Our send recv interface will use `ret` instead of `msg` now? This will require us to change almost all our code... Besides I'm not sure if it is the right thing to do. If some RTL developer is not familiar with method based and just want to write RTL this naming convention can be quite confusing

Reply:
Well sorry I mean enq/deq

Reply:
Send/Recv will still use message. Enq will still use msg because I think directly moving to arg is too aggressive, but using ret for Deq(Give) is what we are doing as the first step. This is to avoid mixing the fields when someone tries to connect two very unrelated interfaces